Formula/postgresql@13.rb

```diff
@@ -22,7 +22,7 @@ class PostgresqlAT13 < Formula
   keg_only :versioned_formula
 
   # https://www.postgresql.org/support/versioning/
-  deprecate! date: "2024-11-13", because: :unsupported
+  deprecate! date: "2025-11-13", because: :unsupported
```

Comment:
This was wrong according to the link.

Reply:
That's the moment it'll stop being supported (`disable!`) though, arguably the deprecation should be before that.

Reply:
As far as I can tell the others have this date set to the upstream deprecation date. Have I missed something here? I may well have 😅

Reply:
You've found a longstanding TODO from me: "allowing disable and deprecate concurrently". Since otherwise we need magic like this.

It's very likely I didn't catch this in the other formula. I'll see if I can make a PR for those.